src/views/home/index.js

```diff
@@ -2,17 +2,20 @@ import React from 'react';
 import Footer from '../../components/footer';
 import Header from '../../components/header';
 import Menu from '../../components/menu';
+import Gallery from '../../components/gallery';
 import './home.css';
 
-function Home() {
+function Home({sessions}) {
+    console.log("Printing session..\n")
     return (
         <div className="Home">
             {/* TODO: HOME VIEW */}
             <Header/>
             <Menu/>
-            <div>
-                TODO: GALLERY COMPONENT
+            <div className="gallery__container">
+                {sessions.map((session, index) =>  <Gallery key={index} snapshots={session.snapshots}/>)}
```

Comment:
Use the `uuidv4()` to set ids in your session and snapshot constructors and then use id for key instead of index. Index makes a bad key as the order in the array can change.